File: AppChat/BackEnd/AppChat/Models/GroupMemberModel.cs
using System.ComponentModel.DataAnnotations;

namespace AppChat.Models
{
    public class GroupMemberModel
    {
        public string? GroupId { get; set; }
        [Required]
        public string UserId
        { get; set; }
        [Required]
        public string Role
            { get; set; }
        }
}

File: AppChat/FontEnd/app/src/main/java/com/example/appchatit/models/GroupMemberModel.java
package com.example.appchatit.models;

import java.io.Serializable;

public class GroupMemberModel implements Serializable {
    private String groupId;
    private String userId;
    private String role;

    public GroupMemberModel() {
    }

    public GroupMemberModel(String groupId, String userId, String role) {
        this.groupId = groupId;
        this.userId = userId;
        this.role = role;
    }

    public String getGroupId() {
        return groupId;
    }

    public void setGroupId(String groupId) {
        this.groupId = groupId;
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public String getRole() {
        return role;
    }

    public void setRole(String role) {
        this.role = role;
    }
}